xapian-applications/omega/templates/query

```diff
@@ -58,7 +58,9 @@ if (!("autofocus" in document.createElement("input")))
  document.getElementById("omega-autofocus").focus();
 </script>
 <input type=submit value="Search">
-$if{,$include{inc/anyalldropbox},$include{inc/anyallradio}}
+${Todo: Replace radios with search buttons}
```

Comment:
Would buttons be the right approach here? Right now, you can toggle between the options. Would you want to replace a single search button with "search" and "search (AND)" or something?